packages/web-react/src/components/UNSTABLE_PartnerLogo/stories/UNSTABLE_PartnerLogo.stories.tsx

```diff
@@ -23,7 +23,7 @@ const meta: Meta<typeof UNSTABLE_PartnerLogo> = {
     },
   },
   args: {
-    hasSafeAreaDisabled: false,
+    hasSafeArea: true,
```

Comment:
this prop should be in argTypes too, please add it there. We missed that before.